core/dbt/task/debug.py

```diff
@@ -125,7 +126,7 @@ def _load_project(self):
             return red('ERROR not found')
 
         try:
-            self.project = Project.from_current_directory(self.cli_vars)
+            self.project = Project.from_project_root(self.project_dir, self.cli_vars)
```

Comment:
Looks like this line is too long per our styleguide:
```
core/dbt/task/debug.py:129:80: E501 line too long (85 > 79 characters)
```

via: https://circleci.com/gh/fishtown-analytics/dbt/19769?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Can you split this onto two lines?